apps/charge-accounts-service/src/operators/operators.service.ts

```diff
@@ -250,6 +253,7 @@ export class OperatorsService {
       // and the webhook call will respond negatively. In such cases, a retry mechanism from the
       // notification service will be utilized.
       await this.updateIsActivated(wallet._id, true)
+      this.analyticsService.operatorAccountActivationEvent({ id: wallet.ownerId, projectId: project._id })
```

Comment:
@NikolayRn as you mentioned about decoupling, are we still going to use the analytics service in the core business operators account service?

> decoupling of analytics logic from core business functionalities